### PR TITLE
Prevent duplicate answer_ids across different list collectors

### PR DESCRIPTION
**Pull request by liamtoozer**
### What is the context of this PR?
Following [a change to schema validation](https://github.com/ONSdigital/eq-questionnaire-validator/pull/228) to prevent duplicating an `answer_id` across different list collectors in a schema, we should update any existing test schemas in Runner that break this validation.

### How to review
Ensure that the schema changed in this PR passes validation successfully

### Checklist
* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by liamtoozer** [comment]
[Bug: Validator allows duplicate answer ids in different list collectors](https://trello.com/c/fB5R54hy/5965-bug-validator-allows-duplicate-answer-ids-in-different-list-collectors)

**Review by VirajP1002 — COMMENTED** [review]
All schemas passed validation and all tests passed! 👍

File: scripts/run_validator.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
-tag=latest
+tag=prevent-duplicate-answer-ids-across-different-list-colectors
```
Comment (liamtoozer) [inline scripts/run_validator.sh:2]: Reminder to revert before merge

**Review by katie-gardner — APPROVED** [review]
Looking good! (Sorry looks like we both edited repeating sections with hub & spoke, your naming convention is better than what I did so I'd suggest just going with 'use mine' during the merge so we keep your naming)

**Comment by liamtoozer** [comment]
> Looking good! (Sorry looks like we both edited repeating sections with hub & spoke, your naming convention is better than what I did so I'd suggest just going with 'use mine' during the merge so we keep your naming)

Ah cool no worries! Will take a look, thanks for the heads up

**Review by berroar — APPROVED** [review]

**Pull request merged**